### PR TITLE
Fe 558 special small button updates

### PR DESCRIPTION
**Pull request by shan8851**
# Description

Fixing issues with special button and adding new `icon only` special button

---

Fixes # (issue)
https://uniwhales.atlassian.net/browse/FE-558?atlOrigin=eyJpIjoiNTg5MjkwYjBmMmIxNGU1ZmJhNzBkNzUxM2I2ZGU1OGEiLCJwIjoiaiJ9

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Test Configuration**:
* Hardware: Android/iphone/macbook/pc/laptop

## Was this feature tested on all the browsers?

- [X] Chrome/Brave
- [X] Firefox
- [] Safari 

# Checklist:

- [X] I have performed a self-review of my code
- [ ] I have tested my changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] Storybook covers all new/pre-existing variations without need to change controls to test it
- [X] I have upgraded version in package.json
- [X] I have assigned QA and Reviewers
- [X] I have labeled the PR accordingly
- [X] I have updated time spent in my jira ticket
- [ ] Build succeeded

---

### CODE REVIEW

File: src/atoms/buttons/button.tsx [hunk 1]
```diff
@@ -225,11 +225,14 @@ const ButtonSpecialSmallNormal = Styled(Button)<Pick<ButtonProps, 'width'>>`
     height: 16px;
     fill: ${({ theme }) => theme.textShades.SHADE_MINUS_3}!important;
   };
-
+  // disabled
   ${(props) => props.disabled && css`
     background: ${props.theme.containerAndCardShades.SHADE_PLUS_1};
-    color: ${props.theme.textShades.SHADE_MINUS_1};
+    color: ${props.theme.textShades.SHADE_MINUS_1}!important;
```
Comment (ArinCantCode) [inline src/atoms/buttons/button.tsx:231]: tbh i think we should double check why we need to force !important here. Can you investigate if we can get rid of this?

**Review by ArinCantCode — CHANGES_REQUESTED** [review]

File: src/atoms/buttons/button.tsx (hunk 1 above)
Comment (shan8851) [reply src/atoms/buttons/button.tsx:231]: yeah there is a lot of important going on in the button file in general, I think on the scg it was before we removed `fill` property from icons, so maybe a lot of the important can be gone. Will spend some time and see what I can do

Comment (shan8851) [reply src/atoms/buttons/button.tsx]: @ArinCantCode I managed to remove a couple and didn't add any additional, but we still need a bunch of !important on the svgs. I guess we could potentially add something to the backlog to take a look another look at this at some point

**Review by goshatravin — APPROVED** [review]

**Comment by Xtoseone** [comment]
I don't know - important or not. a small different size
Скриншот 14-11-2022 125153

**Comment by shan8851** [comment]
@Xtoseone yeah I think this is very small difference and can be dependent on exact sizings of SVG, I think it is better to look at the sizing of the outer container.

Not sure what you think @ArinCantCode @goshatravin

**Review by Xtoseone — APPROVED** [review]

**Pull request merged**